transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -467,6 +470,37 @@ public DefaultHttpSession call() {
                         SESSION_KEY.set(session, newHttpSession);
                         return newHttpSession;
                     }
+
+
+
+                    private IoHandler getIoHandler(DefaultHttpSession newHttpSession, boolean invalidHeaders) {
```

Comment:
I would think it would be better to implement this logic in the decoder, as I would in general expect the decoder to detect badly formatted requests responses.  I'm not overly familiar with where in the Decoder this would go.  Perhaps somewhere near: https://github.com/kaazing/gateway/blob/develop/transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpHeaderDecodingState.java